packages/affiliates/scripts/basic-contract.js

```diff
@@ -0,0 +1,14 @@
+require("dotenv").config();
```

Comment:
can probably ignore this, or could be deleted, but could not run the app using `getWeb3`. would get `UnhandledPromiseRejectionWarning: Error: Invalid JSON RPC response: "" ` which went away once i initialized my own web3.